commands/auth.go

```diff
@@ -136,7 +136,8 @@ func loginCmdF(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	allowInsecure := viper.GetBool("insecure-sha1-intermediate")
+	allowInsecureSHA1 := viper.GetBool("insecure-sha1-intermediate")
+	allowInsecureTLS := viper.GetBool("insecure-tls-version")
```

Comment:
I'm wondering if, instead of a boolean, this should be an enum specifying the minimum version. An enterprise deployment might need TLS 1.1 support but also explicitly not support TLS 1.0. E.g. PCI compliance requires servers to disable TLS 1.0 support. Obviously PCI DSS is not directly applicable to mmctl, but it's indicative that other standards and certifications might have similar requirements.

Not a blocker, but something customers might bring up later if we go with the boolean option.

Reply:
Thanks for the feedback @jupenur sounds reasonable.

Reply:
Discussed this with @mgdelacroix offline and we decided to continue with the current approach. If a requirement shows up we can change it accordingly.